Tema

AlbertoMitroi opened the pull request.

AlbertoMitroi committed:
Add files via upload

AlbertoMitroi committed:
Delete opriribenzinarii.cpp

AlbertoMitroi committed:
Add files via upload

Tenebriso commented on alberto/opriribenzinarii.cpp:
```diff
@@ -0,0 +1,28 @@
+
+Determinarea solutiei:
+- Parcurgem benzinariile
+- Daca nu putem ajunge la o benzinarie, ne intoarcem la cea din urma si alimentam. Practic rezervorul nostru devine 10L + Km la care se afla benzinaria la care alimentam 
```
Capacitatea rezervorului este <max_rez>, iar la fiecare oprire la o benzinărie, se reumple rezervorul (adică rez = max_rez), nu poate reține mai mult de max_rez.

Tenebriso commented on alberto/opriribenzinarii.cpp:
```diff
@@ -0,0 +1,28 @@
+  int b, ben[50], rez, opriri = 0;
+  cout << "Numarul de benzinarii este: "; cin >> b;
+  cout << "Capacitatea rezervorului este: "; cin >> rez;
+  // Am incercat cu int *ben si apoi ben = new int(b); dar daca dau un b mai mare de 7 nu merge
```
probabil crapă din cauza liniei 20, ar trebui `ben = new int(b+1)` dacă începe de la 1 și merge până la `b`

AlbertoMitroi committed:
Modificat

AlbertoMitroi committed:
Add files via upload

pt a 2 a problema nu prea imi mai aduc nimic aminte despre caractere si functiile de caractere.. am cautat pe net dar nu ma descurc cu ele.. nu prea le am inteles nici la clasa

AlbertoMitroi committed:
Add files via upload

AlbertoMitroi committed:
Add files via upload
